package.json

```
@@ -43,8 +43,8 @@
           "name": "Compiler Toolchain"
         },
         {
-          "id": "ExecutorView",
-          "name": "Executor"
+          "id": "TargetDevice",
+          "name": "Target Device"
```

```suggestion
          "name": "Target Devices"
```

I looked into other extensions and they used plural noun.